tests/ShopBundle/Acceptance/acceptance/PageObject/Front/ProductFilterPage.php

```diff
@@ -23,7 +22,11 @@ public function __construct(StrictWebDriver $strictWebDriver, AcceptanceTester $
      */
     public function setMinimalPrice($price)
     {
-        $this->tester->fillFieldByCss('#product_filter_form_minimalPrice', $price . WebDriverKeys::ENTER);
+        /**
+         * The input for minimal price is hidden by design so Codeception is not able to fill it directly.
+         * There is a jQueryUI price slider in the filter form but making Codeception to handle it correctly would be overkill.
+         */
+        $this->tester->executeJS(sprintf('$("#product_filter_form_minimalPrice").val(%d).change()', $price));
```

Comment:
ok. Maybe we could at least check whether the component that handles this event is visible for User.
`%d` -> `%.2f`

Reply:
after discussing it personally we agreed that checking the visibility is not necessary :slightly_smiling_face: 

> %d -> %.2f  :heavy_check_mark: